doc/reference/reference_lua/net_box.rst

```diff
@@ -911,62 +911,65 @@ Here are commands for a quick sandbox setup:
 
 And here starts the example:
 
-.. code-block:: tarantoolsession
+..  code-block:: tarantoolsession
 
     tarantool> net_box = require('net.box')
     ---
     ...
-    tarantool> function example()
-             >   local conn, wtuple
-             >   if net_box.self:ping() then
-             >     table.insert(ta, 'self:ping() succeeded')
-             >     table.insert(ta, '  (no surprise -- self connection is pre-established)')
-             >   end
-             >   if box.cfg.listen == '3301' then
-             >     table.insert(ta,'The local server listen address = 3301')
-             >   else
-             >     table.insert(ta, 'The local server listen address is not 3301')
-             >     table.insert(ta, '(  (maybe box.cfg{...listen="3301"...} was not stated)')
-             >     table.insert(ta, '(  (so connect will fail)')
-             >   end
-             >   conn = net_box.connect('127.0.0.1:3301')
-             >   conn.space.tester:delete({800})
-             >   table.insert(ta, 'conn delete done on tester.')
-             >   conn.space.tester:insert({800, 'data'})
-             >   table.insert(ta, 'conn insert done on tester, index 0')
-             >   table.insert(ta, '  primary key value = 800.')
-             >   wtuple = conn.space.tester:select({800})
-             >   table.insert(ta, 'conn select done on tester, index 0')
-             >   table.insert(ta, '  number of fields = ' .. #wtuple)
-             >   conn.space.tester:delete({800})
-             >   table.insert(ta, 'conn delete done on tester')
-             >   conn.space.tester:replace({800, 'New data', 'Extra data'})
-             >   table.insert(ta, 'conn:replace done on tester')
-             >   conn.space.tester:update({800}, {{'=', 2, 'Fld#1'}})
-             >   table.insert(ta, 'conn update done on tester')
-             >   conn:close()
-             >   table.insert(ta, 'conn close done')
-             > end
+    -- Self connection is pre-established.
+    -- In this case, `conn = net_box.connect('localhost:3301')`
+    -- can be replaced by `conn = net_box.self`.
+    tarantool> conn = net_box.self
```

Comment:
The former example was more extensive - it also demonstrated how `net_box.connect` works.

Anyway, I don't understand what's wrong with the example. Quoting the description of https://github.com/tarantool/doc/issues/2271:

> очень плохой пример, ничего не понятно
> зачем таблица ta?
> почему внутри функции не делать print?

`print()` would print to the log, not to the console so we use a table to return the operation log. There isn't much point in using a global table here - we could turn `ta` into the function-local variable and return it from `example()` directly, which would be more concise.